src/c++/perf_analyzer/genai-perf/tests/test_llm_inputs.py

```diff
@@ -61,15 +61,15 @@ def test_input_type_synthetic_no_tokenizer(self):
         """
         with pytest.raises(GenAIPerfException):
             _ = LlmInputs._check_for_tokenzier_if_input_type_is_synthetic(
-                input_type=PromptSource.SYNTHETIC, tokenizer=None
+                input_type=PromptSource.SYNTHETIC, tokenizer=None  # type: ignore
             )
 
     def test_illegal_starting_index(self):
         """
         Test for exceptions when illegal values are given for starting index
         """
         with pytest.raises(GenAIPerfException):
-            _ = LlmInputs._check_for_valid_starting_index(starting_index="foo")
+            _ = LlmInputs._check_for_valid_starting_index(starting_index="foo")  # type: ignore
```

Comment:
why are we ignoring these types?
Is the general practice to ignore typing on tests?

Reply:
This is a test case of an illegal input, so that's the only way to do it.